.github/workflows/release_mac.yml

```diff
@@ -61,17 +62,33 @@ jobs:
       run: |
         # Install Protobuf from source
         export NUM_CORES=`sysctl -n hw.logicalcpu`
-        source workflow_scripts/protobuf/build_protobuf_unix.sh $NUM_CORES $(pwd)/protobuf/protobuf_install
-        if [ '${{ github.event_name }}' == 'schedule' ]; then
-          sed -i '' 's/name = "onnx"/name = "onnx-weekly"/' 'pyproject.toml'
-          export ONNX_PREVIEW_BUILD=1
-        fi
+        source workflow_scripts/protobuf/build_protobuf_unix.sh $NUM_CORES $(pwd)/protobuf/protobuf_install    
         python -m build --wheel
 
-    - uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+    - name: Build wheel (schedule)
```

Comment:
Curious why we need two different steps for scheduled runs / non scheduled runs?

Reply:
It is not really necessary, I withdraw the change.  I thought it is relatively little code, and you avoid the if queries in the shellcode but solve everything via if in the jobs and steps directly from github